### DISCUSSION TITLE
[Help] TreePicker - make last child selectable only

### DISCUSSION THREAD

**Discussion by johnboncalos** [rsuite/rsuite#2687] [Q&A]
Hi new here and thanks for this awesome component! I have a category that has 3 levels (Computers->Graphics->XFX) and I only want the user to be able to select XFX category. I've combed over the options and nothing there that can make this happen. I appreciate your replies.

**Comment by simonguo** [comment]
I guess `Cascader` component is more suitable for your application scenario? 
https://rsuitejs.com/components/cascader/
